src/components/documentViewer/utils/index.js

```diff
@@ -18,8 +21,8 @@ export const documentTemplates = (document, handleHeightUpdate) => {
   return [...selectedTemplate, ...templatesFromAttachments];
 };
 
-export const documentTemplateTabs = document => {
-  const templates = documentTemplates(document);
+export const documentTemplateTabs = (document, templateRegistry) => {
+  const templates = documentTemplates(document, () => void 0, templateRegistry);
```

Comment:
This is the first time I've seen `void 0`. May I know what's the different between this and `() => undefined`? Also, there might be a need for `handleHeightUpdate` to be passed into the template registry. For instance, if the component is lazy loaded, it wont know it's height until it was later updated and the DOM may not be re-rendered because the prop of this or it's parent component may not have changed.